app/src/test/java/com/github/se/signify/model/home/quest/FirestoreQuestRepositoryTest.kt

```diff
@@ -43,7 +43,7 @@ class QuestRepositoryFirestoreTest {
   private lateinit var mockAuth: FirebaseAuth
   private lateinit var mockUser: FirebaseUser
 
-  private lateinit var questRepositoryFirestore: QuestRepositoryFireStore
+  private lateinit var firestoreQuestRepositoryFirestore: FirestoreQuestRepository
```

Comment:
I think here you wanted to write firestoreQuestRepository
```suggestion
  private lateinit var firestoreQuestRepository: FirestoreQuestRepository
```

Reply:
You're right.
Done.